wake-lock/wakelock-disabled-by-feature-policy.https.sub.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<body>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/feature-policy/resources/featurepolicy.js></script>
+<script>
+  'use strict';
+
+  const same_origin_src = '/feature-policy/resources/feature-policy-wakelock.html';
+  const cross_origin_src = 'https://{{domains[www]}}:{{ports[https][0]}}' +
+      same_origin_src;
+  const header = 'Feature-Policy header {"wake-lock" : []}';
+
+  promise_test(() => {
```

Comment:
switch to async, and make this a try/catch, with the error being `unreached_assertion()`.

Reply:
I would prefer `promise_rejects` if we know what the error should be.

Reply:
Yeah, good point. I agree.